learnosity_sdk/request/__init__.py

```diff
@@ -0,0 +1,5 @@
+
+__all__ = ["Init", "DataApi"]
+
+from .init import Init
+from .dataapi import DataApi
```

Comment:
I never saw this syntax (leading dot) before. Does it have a special meaning?

Reply:
I believe it's shorthand for importing something from learnosity_sdk/request (ie. the parent module). I took it from here https://www.safaribooksonline.com/library/view/python-cookbook-3rd/9781449357337/ch10s04.html

Reply:
Does it work without the leading `.`?

Reply:
For completeness: We need those imports so the `request` module actually has first-class objects `Init` and `DataApi`, as described by the `__all__` array.